content/en/docs/installation/kubectl.md

```diff
@@ -0,0 +1,95 @@
+---
+title: "Kubectl apply"
+linkTitle: "Kubectl apply"
+weight: 20
+type: "docs"
+---
+
+> Note: From cert-manager `v1.2.0` onward, the minimum supported version of
+> Kubernetes is `v1.16.0`. Users still running Kubernetes `v1.15` or below should
+> upgrade to a supported version before installing cert-manager.
```

Comment:
note: as with the helm doc, I think there are just too many notes here, but I've made enough suggestions already with the helm doc, so I'm not going to go through here and suggest changes :joy:  We should revisit and reword a lot of this but I don't think we need to do that in this PR.